### PR TITLE
WIP: added e2e test case

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Experimental for functional test case.

#### Why?

A pure functional test case would be awesome.

#### Example Usage

TODO

 - [ ] Integrate Test which will create a new page and call the page on the website
 - [ ] Move install symfony/panther to CI as it only will work with symfony 4

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -27,14 +26,14 @@
     }
 }
 
-require __DIR__ . '/../vendor/autoload.php';
+require __DIR__.'/../vendor/autoload.php';
```
Comment (alexander-schranz) [inline public/index.php]: is moved to own PR: https://github.com/sulu/sulu/pull/4308

File: public/index.php [hunk 2]
```diff
@@ -54,18 +53,24 @@
     Request::setTrustedHosts(explode(',', $trustedHosts));
 }
 
-$kernel = new Kernel($env, $debug, SuluKernel::CONTEXT_WEBSITE);
+$suluContext = SuluKernel::CONTEXT_WEBSITE;
```
Comment (alexander-schranz) [inline public/index.php]: is moved to own PR: https://github.com/sulu/sulu/pull/4308

File: tests/Functional/WebsiteTest.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        $crawler = $client->request('GET', '/admin');
+
+        $this->assertContains('SULU', $crawler->filter('title')->html());
+
+        $client->waitFor('form');
+
+        $crawler->selectButton('button:last-child')->
+    }
+
+    protected static function createKernel(array $options = [])
```
Comment (alexander-schranz) [inline tests/Functional/WebsiteTest.php]: should be moved to SuluTestBundle

File: tests/Functional/E2ETest.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        $client = static::createPantherClient();
+        $crawler = $client->request('GET', '/admin');
+
+        $this->assertContains('SULU', $crawler->filter('title')->html());
+
+        $client->waitFor('form');
+
+        $crawler->selectButton('button:last-child')->form([
+            '_username' => 'admin',
```
Comment (alexander-schranz) [inline tests/Functional/E2ETest.php]: currently there exists no names on the inputs which make e2e tests really hard /cc @danrot @chirimoya

Comment (danrot) [reply tests/Functional/E2ETest.php]: Feel free to add them :wink:
